docs/js/getting-started.md

```diff
@@ -20,53 +20,296 @@ import LicenseBadge from '../../src/sap/sdk-js/LicenseBadge'
+| ✅ Init finished successfully.                                |
+|                                                               |
+| 🚀 Next steps:                                                |
+| - Run the application locally (`npm run start:dev`)           |
+| - Deploy your application (`npm run deploy`)                  |
+|                                                               |
+| 🔨 Consider setting up Jenkins to continuously build your app |
+| Use `sap-cloud-sdk add-cx-server` to create the setup script  |
++---------------------------------------------------------------+
+```
+
+## Project files and folders
+<!--not sure about this part-->
+
+The project contains the following files and folders, among others, to get you started with the SAP Cloud SDK for JavaScript:
+
+#### NPM / Project
+
+- **`package.json`**: Specifies dependencies, metadata and user-defined scripts. The application comes with some predefined scripts and dependencies, that will be explained in detail in the course of this group of tutorials.
+- **`.npmrc`**: The **`npm`** configuration file. The SAP Cloud SDK consists of some generic libraries, that are available as Open Source Software and service libraries for the whitelisted SAP S/4HANA APIs, referred to as the Virtual Data Model (VDM). In the scaffolding we specify the registry for the `@sap` scope, where the VDM libraries are published.
+
+#### TypeScript
+
+- **`tsconfig.json`**: Configuration file for `TypeScript`. This is not needed in the plain `JavaScript` version.
+- **`tslint.json`**: Configuration file for `tslint`, the de facto default linter for `TypeScript`.
+
+#### Continuous Delivery
+
+- **`Jenkinsfile`**: Jenkins pipeline definition file for quality assurance. It uses the [SAP Cloud SDK's Continuous Delivery Toolkit](https://github.com/SAP/cloud-s4-sdk-pipeline).
+- **`pipeline_config.yml`**: Pipeline configuration file for the Jenkins pipeline.
+
+#### Cloud Foundry
+
+- **`manifest.yml`**: The deployment descriptor file for `Cloud Foundry in SAP Cloud Platform`.
+
+#### Local development
+
+- **`src/`**: Source code for the initial application.
+
+#### SDK specific
+
+- **`systems.json`+`credentials.json`**: Allows you to maintain destinations for testing purposes.
+- **`sap-cloud-sdk-analytics.json`**: Only if you have agreed to usage analytics during the initialization of your project. You can find more information about anonymous usage analytics [in the CLI's repository](https://github.com/SAP/cloud-sdk-cli/blob/master/usage-analytics.md).
+
+
+## Run the project
+
+To run the application locally, simply run the following command:
+
+```shell
+npm run start:dev
+```
+
+Go to `http://localhost:3000` and you should get a `Hello, World!` in response. Open `src/main.ts` and switch the port from `3000` to `8080`. The corresponding line should then look like this:
+
+```ts
+await app.listen(process.env.PORT || 8080);
+```
```

Comment:
We should explain why (because this will automatically be started using port 8080 in CF). But I think `process.env.PORT` already fixes this, so this step is not needed anymore
@florian-richter Can you confirm that?
@CharlesDuboisSAP Maybe you can double check by trying to push an app without changing this.